targets/targets.json

```diff
@@ -5271,6 +5271,60 @@
         "device_name": "STM32L433RC",
         "bootloader_supported": true
     },
+    "NUCLEO_L452RE_P": {
+        "inherits": [
+            "FAMILY_STM32"
+        ],
+        "supported_form_factors": [
+            "ARDUINO",
+            "MORPHO"
+        ],
+        "core": "Cortex-M4F",
+        "extra_labels_add": [
+            "STM32L4",
+            "STM32L452",
+            "STM32L452xx",
+            "STM32L452xE",
+            "STM32L452RE"
```

Comment:
STM32L452, STM32L452xx and STM32L452RE not needed ?

Reply:
Which of these lines (any of the target.json lines), if any, are required for adding the `"bootloader_supported": true` feature in order to make it work?

Incidentally, when I did have that one added, the mbed script would complain that it could not find STM32L452RE in some file. This may be by design, but it was beyond my understanding either way.

Reply:
"bootloader_supported" needs the correct device_name value, see below comment